### PR TITLE
fix pp parser

### PR DESCRIPTION
**Pull request by cpignedoli**
The empa parser for pp has to be updated according to the changes in the aiida-quantumespresso parser. parsing of cube files was affected

### CODE REVIEW

**Comment by AndresOrtegaGuerrero** [comment]
I think we should fix the ci/test perhaps a test could be added for this parser.

File: aiida_nanotech_empa/parsers/pp_parser.py [hunk 1]
```diff
@@ -27,7 +28,7 @@ def parse_gaussian(self, data_file_str):
         voxel_array = cube.cell / np.atleast_2d(cube.data.shape).T
         numbers = cube.ase_atoms.numbers
 
-        data_units = self.units_dict[self.output_parameters["plot_num"]]
+        # data_units = self.units_dict[self.output_parameters["plot_num"]]
```
[the remark this answers is not in the document]
Comment (AndresOrtegaGuerrero) [reply aiida_nanotech_empa/parsers/pp_parser.py:31]: Now is clear!

**Review by AndresOrtegaGuerrero — APPROVED** [review]
Looks fine to me!

**Comment by yakutovicha** [comment]
I am going to merge this and release a new beta version of the package. The `aiida-quantumespresso<4.3` constraint should be removed for the final release.

**Pull request merged**